sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -56,8 +57,13 @@ def test_dict_learning_overcomplete():
     assert dico.components_.shape == (n_components, n_features)
 
 
-# positive lars deprecated 0.22
-@pytest.mark.filterwarnings('ignore::DeprecationWarning')
+def test_dict_learning_lars_positive_parameter():
+    n_components = 5
+    alpha = 1
+    assert_raises(ValueError, dict_learning, X, n_components,
```

Comment:
We are using `pytest.raises` form now on. You should also match the error message.